package.json

```diff
@@ -52,6 +52,7 @@
     },
     "dependencies": {
         "@0glabs/0g-ts-sdk": "0.2.1",
+        "@ai16z/adapter-postgres": "0.1.6-alpha.4",
```

Comment:
there has to be a better way to do this than pinning a specific version, even latest would be an improvement

Reply:
actually why is this here?